tests/deploy_test.go

```diff
@@ -13,6 +14,75 @@ import (
 
 var emptyQueryString = ""
 
+func Test_Scaling_Up(t *testing.T) {
+	functionRequest := requests.CreateFunctionRequest{
+		Image:      "functions/alpine:latest",
+		Service:    "test-scaling",
+		Network:    "func_functions",
+		EnvProcess: "env",
+	}
+
+	deployStatus, deployErr := deploy(t, functionRequest)
+	if deployErr != nil {
+		t.Errorf(deployErr.Error())
+		t.Fail()
+		return
+	}
+
+	if deployStatus != http.StatusOK && deployStatus != http.StatusAccepted {
+		t.Errorf("got %d, wanted %d or %d", deployStatus, http.StatusOK, http.StatusAccepted)
+	}
+
+	invoke(t, functionRequest.Service, emptyQueryString, http.StatusOK)
```

Comment:
Doesn't invoke return anything? What if it was an error?

Reply:
`invoke()` returns a slice of bytes. Errors are handled by the function invokeWithVerb(). As for why it was done this way, I'd have to ask you (at least you a year ago).
```
func invoke(t *testing.T, name string, query string, expectedStatusCode ...int) []byte {
	return invokeWithVerb(t, http.MethodPost, name, query, expectedStatusCode...)
}
```

Reply:
It would be nice if we could bubble it up in a follow-up PR to handle the error at the top level.

Reply:
I agree--should I make an issue for it?